lib/al/Library/Camera/SnapShotCameraCtrl.cpp

```diff
@@ -1,16 +1,38 @@
 #include "SnapShotCameraCtrl.h"
-#include <al/Library/Yaml/ByamlUtil.h>
+
+namespace {
+using namespace al;
+NERVE_IMPL(SnapShotCameraCtrl, Wait);
+NERVE_IMPL(SnapShotCameraCtrl, Reset);
+
+struct {
+    NERVE_MAKE(SnapShotCameraCtrl, Reset);
+    NERVE_MAKE(SnapShotCameraCtrl, Wait);
+} NrvSnapShotCameraCtrl;
+}  // namespace
 
 namespace al {
+
+#ifdef NON_MATCHING
+SnapShotCameraCtrl::SnapShotCameraCtrl(const al::SnapShotCameraSceneInfo* info) : al::NerveExecutor("スナップショットモード中のカメラ制御") {
+    mCameraSceneInfo = info;
+    initNerve(&NrvSnapShotCameraCtrl.Reset, 0);
+
+    al::SnapShotCameraCtrlParam* param = new al::SnapShotCameraCtrlParam();
+    mParam = param;
+}
+#endif
+
 #ifdef NON_MATCHING
 void SnapShotCameraCtrl::load(const al::ByamlIter& iter) {
-    al::ByamlIter param;
-    if (!al::tryGetByamlIterByKey(&param, iter, "SnapShotParam"))
+    auto param = mParam;
+    al::ByamlIter paramIter;
+    if (!al::tryGetByamlIterByKey(&paramIter, iter, "SnapShotParam"))
         return;
-    if (al::tryGetByamlF32(&mParam->mMinFovyDegree, param, "MinFovyDegree"))
-        mParam->gotMin = true;
-    if (al::tryGetByamlF32(&mParam->mMinFovyDegree, param, "MinFovyDegree"))
-        mParam->gotMax = true;
+    if (al::tryGetByamlF32(&mParam->mMinFovyDegree, paramIter, "MinFovyDegree"))
+        param->gotMin = true;
+    if (al::tryGetByamlF32(&mParam->mMaxFovyDegree, paramIter, "MaxFovyDegree"))
+        param->gotMax = true;
```

Comment:
What/Why did you change this, if it's still mismatching - does it give a better result than before?

Reply:
The logic before was flat out wrong, checking for `MinFovyDegree` twice. I fixed that but it's still mismatching.

Reply:
Got this one to match- resolving this one